Examples/CppCMake/Documentation.rst

```diff
@@ -17,86 +19,105 @@ Here is our SimpleITK C++ example file **sitk_example.cxx**.
    :lines: 18-
 
 
-Invoking **cmake** as below in the same directory as these files will create the project buildsystem::
+Invoking **cmake** as below in the same directory as these files will create
+the project buildsystem::
 
     cmake .
 
 Required CMake Variables
 ########################
 
-Running CMake will produce a **CMakeCache.txt** file that is the CMake configuration.  Within that
-text file there are two variables needed to build a SimpleITK program, *SimpleITK_DIR* and *ITK_DIR*.
-SimpleITK_DIR needs to be set to the directory that contains the file SimpleITKConfig.cmake, and
-ITK_DIR needs to be the directory containing the file ITKConfig.cmake.
+Running CMake will produce a **CMakeCache.txt** file that is the CMake
+configuration.  Within that text file there are two variables needed to build
+a SimpleITK program, **SimpleITK_DIR** and **ITK_DIR**.  SimpleITK_DIR needs to
+be set to the directory that contains the file :code:`SimpleITKConfig.cmake`,
+and ITK_DIR needs to be the directory containing the file
+:code:`ITKConfig.cmake`.
 
-CMake attempts to find these files automatically, but it is unlikely to find them unless SimpleITK
-and ITK are installed into the system.
+CMake attempts to find these files automatically, but it is unlikely to find
+them unless SimpleITK and ITK are installed into the system.
 
-If SimpleITK has been built using the SuperBuild process, as described in :doc:`building`, then
-these required directories can be found in the SuperBuild location.  The SimpleITK_DIR variable
-should refer to the directory SimpleITK-build within the SuperBuild location, and the ITK_DIR
-variable should refer to the directory ITK-build in the SuperBuild location.
+If SimpleITK has been built using the SuperBuild process, as described in
+:doc:`building`, then these required directories can be found in the SuperBuild
+location.  The SimpleITK_DIR variable should refer to the directory
+SimpleITK-build within the SuperBuild location, and the ITK_DIR variable should
+refer to the directory ITK-build in the SuperBuild location.
 
 Here are what the settings in a CMakeCache.txt file should look like::
 
     //The directory containing a CMake configuration file for ITK.
-    ITK_DIR:PATH=/home/dchen/SuperBuild/ITK-build
+    ITK_DIR:PATH=/home/dchen/SimpleITK-build/ITK-build
 
```

Comment:
Possibly replace "/home/dchen/" with the more generic "path_to_simpleitk_build_directory/"?